externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/spark/ui/EnginePage.scala

```diff
@@ -129,102 +158,199 @@ case class EnginePage(parent: EngineTab) extends WebUIPage("") {
     }
   }
 
-  /** Generate stats of statements for the engine */
-  private def generateStatementStatsTable(request: HttpServletRequest): Seq[Node] = {
-
-    val numStatement = store.getStatementList.size
-
-    val table =
-      if (numStatement > 0) {
+  /** Generate stats of running statements for the engine */
+  private def generateStatementStatsTable(
+      request: HttpServletRequest,
+      running: Seq[SparkOperationEvent],
+      completed: Seq[SparkOperationEvent],
+      failed: Seq[SparkOperationEvent]): Seq[Node] = {
+
+    val content = mutable.ListBuffer[Node]()
+    if (running.nonEmpty) {
+      val sqlTableTag = "running"
+      val table =
+        statementStatsTable(request, sqlTableTag, parent, running)
+      content ++=
+        <span id="running" class="collapse-aggregated-runningSqlstat collapse-table"
```

Comment:
should the element id be unique globally on the whole page? maybe `running-sqlstat` is better